packages/core/src/browser/style/dialog.css

```
@@ -105,3 +105,25 @@
     font-size: var(--theia-ui-font-size1);
     display: block;
 }
+
+.theia-DialogNode{
+    line-height: var(--theia-content-line-height);
+    margin-top: calc(var(--theia-ui-padding)*1.5);
+    margin-left: calc(var(--theia-ui-padding)*2.5);
+}
```

[minor] the styling of css classes can be improved for consistency and readability:

```suggestion
.theia-DialogNode {
    line-height: var(--theia-content-line-height);
    margin-top: calc(var(--theia-ui-padding)*1.5);
    margin-left: calc(var(--theia-ui-padding)*2.5);
}
```

---

thank you for your review @vince-fugnitto. I have updated the code with this change in mind.